google-cloud-spanner/lib/google/cloud/spanner/service.rb

```diff
@@ -259,6 +259,18 @@ def create_session database_name, labels: nil
           end
         end
 
+        def batch_create_sessions database_name, session_count, labels: nil
+          opts = default_options_from_session database_name
+          session = Google::Spanner::V1::Session.new labels: labels if labels
+          execute do
+            # The response may have fewer sessions than requested in the RPC.
```

Comment:
Isn't this handled by the method being called?

Reply:
No, in fact it is handled in the higher-level class that calls the method here. (`Client#batch_create_new_sessions`). At this location, the comment is correct.